build.gradle

```diff
@@ -1,4 +1,5 @@
 plugins {
+    id "com.dorongold.task-tree" version "2.1.1"
```

Comment:
What does this do?

Reply:
Oops, didn't mean to include that but actually - its pretty nice for troubleshooting gradle dependencies - specifically it shows what dependencies TASKS have, e.g.:

```
ubuntu@ip-172-31-18-157:~/git/opensearch-migrations$ ./gradlew -p RFS assemble taskTree

> Task :RFS:taskTree

------------------------------------------------------------
Project ':RFS'
------------------------------------------------------------

:RFS:assemble
+--- :RFS:distTar
|    +--- :commonDependencyVersionConstraints:jar
|    |    +--- :commonDependencyVersionConstraints:classes
|    |    |    +--- :commonDependencyVersionConstraints:compileJava
|    |    |    \--- :commonDependencyVersionConstraints:processResources
|    |    \--- :commonDependencyVersionConstraints:compileJava *
|    +--- :coreUtilities:jar
|    |    +--- :coreUtilities:classes
|    |    |    +--- :coreUtilities:compileJava
|    |    |    |    +--- :commonDependencyVersionConstraints:compileJava *
|    |    |    |    \--- :coreUtilities:generateEffectiveLombokConfig
|    |    |    \--- :coreUtilities:processResources
|    |    \--- :coreUtilities:compileJava *
|    +--- :RFS:jar
|    |    +--- :RFS:classes
|    |    |    +--- :RFS:compileJava
|    |    |    |    +--- :commonDependencyVersionConstraints:compileJava *
|    |    |    |    +--- :coreUtilities:compileJava *
|    |    |    |    \--- :RFS:generateEffectiveLombokConfig
|    |    |    \--- :RFS:processResources
|    |    \--- :RFS:compileJava *
|    \--- :RFS:startScripts
|         +--- :commonDependencyVersionConstraints:jar *
|         +--- :coreUtilities:jar *
|         \--- :RFS:jar *
+--- :RFS:distZip
|    +--- :commonDependencyVersionConstraints:jar *
|    +--- :coreUtilities:jar *
|    +--- :RFS:jar *
|    \--- :RFS:startScripts *
\--- :RFS:jar *


(*) - subtree omitted (printed previously)
Add --repeat to allow printing a subtree of the same task more than once.

To see task dependency tree for a specific task, run gradlew <project-path>:<task> <project-path>:taskTree [--depth <depth>] [--with-inputs] [--with-outputs] [--repeat]
Executions of all tasks except for taskTree are skipped. They are used for building the task graph only.
For example, try running gradlew :RFS:build :RFS:taskTree

Deprecated Gradle features were used in this build, making it incompatible with Gradle 9.0.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

See https://docs.gradle.org/8.0.2/userguide/command_line_interface.html#sec:command_line_warnings
```

Reply:
I'm going to keep it in - if anyone would prefer we remove it happy to do so in a follow up PR